### PR TITLE
Feature/v1.1 fix build

### PR DESCRIPTION
**Pull request by cimadai**
I separated return type from model type.

This commit enable that `make goa && make build`.

### CODE REVIEW

File: design/description.go [hunk 1]
```diff
@@ -6,8 +6,8 @@ const (
 	descriptionSignature                       = "The signature, which can be verified with pubkey"
 	descriptionTimestamp                       = "Transaction timestamp"
 	descriptionPubkey                          = "Account Public Key"
-	descriptionSignatories                     = "Account Signatories"
-	descriptionQuorum                          = "最低限必要な数" // TODO
+	descriptionSignatories                     = "Account signatories."
+	descriptionQuorum                          = "Minimum number of signatures required to issue valid transaction."
```
Comment (upamune) [inline design/description.go:10]: Thank you for the translation 😄

**Comment by upamune** [comment]
I do not understand why the "Response" is required for the media type. For example, these are samples of goa. https://github.com/goadesign/goa-cellar/blob/master/design/media_types.go#L52 https://github.com/goadesign/goa-cellar/blob/master/design/media_types.go#L9

Would you please let me know if there is any reason. 
@cimadai

**Comment by cimadai** [comment]
@upamune
Because same type name in media_tpes and user_types.

For example,
[accounts.go#L.223](https://github.com/soramitsu/iroha-gateway/pull/2/files#diff-64fe26f1eef1654992dfb078ce363b84R223) Type("Account"...) is compiled into user_types.go's [`type Account struct {...}`](https://github.com/soramitsu/iroha-gateway/pull/2/files#diff-a4d8cc8c167db024dc93264564da3c28L20) and 
[accounts.go#L.267](https://github.com/soramitsu/iroha-gateway/pull/2/files#diff-64fe26f1eef1654992dfb078ce363b84L267) MediaType("application/vnd.accountResponse+json"...) is compiled into media_types.go's [`type Account struct {...}`](https://github.com/soramitsu/iroha-gateway/pull/2/files#diff-41a3dcbed1eaa90614a4cf835b1fa776L72).

There is same name, but Type("Account") is model type and MediaType("...") is return type.

So I added `Response` to MediaType.

I think the URL you pointed is OK because it does not use same name in Type and MediaType.

**Comment by upamune** [comment]
OK, I see.

But, `Messageresponse` , `Currencyresponse` ...  is a bit storane. 
I have an idea. 
(ex. Account)
```
// NOW
var Account = Type("Account", func() {
...
}
var AccountResponse = MediaType("application/vnd.accountResponse+json", func() {
...
})
var AccountsResponse = MediaType("application/vnd.accountsResponse+json", func() {
...
})

// CHANGE
var AccountPayload = Type("AccountPayload", func() {
...
}
var Account = MediaType("application/vnd.account+json", func() {
...
})
var Accounts = MediaType("application/vnd.accounts+json", func() {
...
})
```

How about this? 
I changed UserType instead of MediaType.  MediaTypes looks natural. ( `application/vnd.account+json` , `application/vnd.accounts+json` )

**Comment by upamune** [comment]
But this way has to change some codes. 
For example `ArrayOf(Account)` → `ArrayOf(AccountPayload)` 
https://github.com/cimadai/iroha-gateway/blob/64d710c160088f9d92b30443dce58feddc449e08/design/accounts.go#L282

**Comment by upamune** [comment]
The goa sample application seems to use the name Payload for UserTypes.  https://github.com/goadesign/goa-cellar/blob/master/design/user_types.go#L10

**Comment by cimadai** [comment]
@upamune Thank you for your advice.
I changed my code with `Payload`.

**Comment by upamune** [comment]
@cimadai Thanks a lot! 

Merged 😄

**Pull request merged**